Payments/Overview/01_General.md

```diff
@@ -0,0 +1,20 @@
+# General information
+The automatic processing and booking of payments are important success and efficiency factors for companies. This is especially true if you receive orders from several sales channels and offer different payment methods via external service providers. By linking the *Payment Processing* and *Payments* modules, the *Actindo Core1 Platform* enables a holistic automated process flow from the receipt of payments to the final allocation of incoming payments. The *Payments* Module acts as a link between payment providers and an accounting solution.      
+
+
+This documentation refers to the *Payments* module version 4.1.0. There may be differences in the user interface, features, and operation if your *Payments* module is installed in a different version.
+
+
+
+## Benefits
+The Actindo *Payment* module offers the following benefits:
+- Automated processing of incoming payments, refunds, and cancellations.
+- Predefined interfaces for connecting to various payment service providers (PSPs) of your choice.
+- Receiving automated revenues and refunds. 
+
+<!---Stefan, welche Modulversionen werden gebraucht?-->
+## Prerequisites
+- The *DataHub* module version XX or higher is installed.
+- The *Global Driver Management* module version XX or higher is installed.
+- If in use, the *Order Management* module version XXX or higher is installed.
+- If in use, the *Payment Processing* module version XXX or higher is installed. 
```

Comment:
```suggestion
- If in use, the *Payment processing* module version XXX or higher is installed. 
```
Payment processing/Zahlungsabwicklung steht nicht in den Abhängigkeiten, also ist 
 es vermutlich nicht zwingend notwendig. Order management/Fakturierung ist aber schon aufgeführt, also "If in use" würde ich weglassen.